src/sidebar/components/search-status-bar.js

```diff
@@ -1,17 +1,21 @@
 'use strict';
 
-const memoize = require('../util/memoize');
+const { createElement } = require('preact');
+const propTypes = require('prop-types');
 
+const memoize = require('../util/memoize');
+const { withServices } = require('../util/service-context');
 const uiConstants = require('../ui-constants');
+const useStore = require('../store/use-store');
 
-// @ngInject
-function SearchStatusBarController(store, rootThread) {
-  this.TAB_ANNOTATIONS = uiConstants.TAB_ANNOTATIONS;
-  this.TAB_NOTES = uiConstants.TAB_NOTES;
-  this.TAB_ORPHANS = uiConstants.TAB_ORPHANS;
-
+function SearchStatusBar({
+  selectedTab,
+  totalAnnotations,
+  totalNotes,
+  rootThread,
+}) {
   const thread = () => {
-    return rootThread.thread(store.getState());
```

Comment:
Hooks have to be used at the top-level of the component function and not inside any conditionals or loops. See [this page of the React docs](https://reactjs.org/docs/hooks-overview.html#rules-of-hooks). The reason is because hooks rely on the order in which they are called being consistent across renders in order to remember their state from the last render.